mdx-web/src/main/java/com/mx/path/model/mdx/web/PathTools.java

```diff
@@ -38,6 +38,15 @@ public class PathTools {
     AUTHENTICATION_REQUIRED_PATHS.add(Pattern.compile("^\\/[^\\/]+\\/accounts\\/.*\\/transactions$").asPredicate());        // MDX OnDemand Accounts
   }
 
+  /**
+   * Detect all paths that require a user.
+   */
+  private static final List<Predicate<String>> USER_REQUIRED_PATHS = new ArrayList<>();
+
+  static {
+    USER_REQUIRED_PATHS.add(Pattern.compile("^\\/[^\\/]+\\/users\\/[^\\/]+\\/.*$").asPredicate()); // All user-scope routes
```

Comment:
I don't think we need this. We could simplify to just have the PATH_USER_ID_PATTERN and check to see if it matches. Basically the same thing, but only one regex evaluation needed.